packages/tracing/src/transaction.ts

```diff
@@ -80,7 +80,8 @@ export class Transaction extends SpanClass implements TransactionInterface {
     // parameterized by virtue of having no parameters in its path
     if (name !== this.name || source !== this.metadata.source) {
       this.metadata.changes.push({
-        source,
+        // log previous source
+        source: this.metadata.source || source,
```

Comment:
Idk about this fallback 🤔 I know this is probably just to make TS happy since the source in metadata might be undefined but in reality, it always has a source. Should we maybe add a comment explaining this?

Or shouldn't the fallback be `"custom"`?

Reply:
you're right - I changed it so that the transaction always defaults to `custom`, unless explicitly set in metadata.